.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,10 +18,7 @@
 - [ ] My name is on the [list of contributors](https://github.com/glotzerlab/signac/blob/master/contributors.yaml).
 - [ ] My code follows the [code style guideline](https://github.com/glotzerlab/signac/blob/master/CONTRIBUTING.md#code-style) of this project.
 - [ ] The changes introduced by this pull request are covered by existing or newly introduced tests.
-
-If necessary:
-- [ ] I have updated the API documentation as part of the package doc-strings.
-- [ ] I have created a separate pull request to update the [framework documentation](https://docs.signac.io/) on [signac-docs](https://github.com/glotzerlab/signac-docs) and linked it here.
+- [ ] I have updated the API documentation as part of the package doc-strings, or this is not necessary.
```

Comment:
As a reviewer I think the “or not necessary” language is potentially unclear. “Not necessary” is sometimes a subjective decision, whereas the existing criteria of updating docs/tests is more objective (did or did not change). Is there a way to make all the criteria more objective?

Reply:
I think we can completely drop this item, because the presence and correct formatting of doc-strings should ideally be enforced by a linter. There is simply no scenario where checking this box is either not necessary or provides significant value IMO. The same is not true for the next item (see comment below).